pallas-crypto/src/kes/mod.rs

```diff
@@ -0,0 +1,128 @@
+use kes_summed_ed25519::kes::{Sum6Kes, Sum6KesSig};
+use kes_summed_ed25519::traits::{KesSig, KesSk};
+use kes_summed_ed25519::PublicKey;
+use thiserror::Error;
+
+/// KES error
+#[derive(Error, Debug)]
+pub enum Error {
+    #[error("KES error: {0}")]
+    Kes(#[from] kes_summed_ed25519::errors::Error),
+}
+
+/// KES secret key
+pub struct KesSecretKey<'a> {
```

Comment:
Is there any scrubbing mechanism in the upstream repo that we can use on Drop to ensure we clear private key data from memory?

Reply:
Yes. I added this here: https://github.com/txpipe/kes/pull/2